CIP-0PRS/CIP-0PRS.lagda.md

```diff
@@ -0,0 +1,1355 @@
+
+## Use Cases
+
+Main benefit is that Peras drastically decreases the need to wait for confirmation for transactions, from minutes/hours to seconds/minutes
+
+* I would highlight Peras making Cardano more attractive option for interoperability solutions like LayerZero, Axelar, WormHole, etc. Maybe we can even go as far to state that Peras is a pre-requisite for this? 
+* Sidechains
+* Exchanges?
+* DApps/service providers, basically anyone accepting or using ADAs, and anyone building some service on top of cardano with some kind of sensitivity to time
+
+## Attacks and Mitigations
+
+
+## Resource Requirements
+
+In this section, we evaluate the impact on the day-to-day operations of the Cardano network and cardano nodes of the deployment of Peras protocol, based on the data gathered over the course of project.
+
+In this section, we evaluate the impact on the day-to-day operations of the Cardano network and cardano nodes of the deployment of Peras protocol, based on the data gathered over the course of project.
+
+### Network
+
+#### Network traffic
+
+For a fully synced nodes, the impact of Peras on network traffic is modest:
+
+* For votes, assuming $U \approx 100$, a committee size of 2000 SPOs, a single vote size of 700 bytes, means we will be adding an average of 14 kB/s to the expected traffic to each node,
+* For certificates, assuming an average of 50 kB size (half way between Mithril and ALBA sizes) means an negligible increase of 0.5 kB/s on average. Note that a node will download either votes or certificate for a given round, but never both so these numbers are not cumulative.
+
+A non fully synced nodes will have to catch-up with the _tip_ of the chain and therefore download all relevant blocks _and_ certificates. At 50% load (current [monthly load](https://cexplorer.io/usage) is 34% as of this writing), the chain produces a 45 kB block every 20 s on average. Here are back-of-the-napkin estimates of the amount of data a node would have to download (and store) for synchronizing, depending on how long it has been offline:
+
+| Time offline | Blocks (GB) | Certificates (GB) |
+|--------------|-------------|-------------------|
+| 1 month      | 5.56        | 1.23              |
+| 3 months     | 16.68       | 3.69              |
+| 6 months     | 33.36       | 7.38              |
+
+#### Network costs
+
+![Typical node inbound & outbound traffic](https://peras-staging.cardano-scaling.org/assets/images/node-average-traffic-49a7cbcee8e4f05a7a42b61e445e5db9.jpg#scale50)
```

Comment:
The link returns 401 Authorization Required

Reply:
Thanks. We're in the process of migrating the diagrams directly into the folder for this CIP.

Reply:
URL fixed.